components/Coder.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
`Coder` seems kinda confusing to me, how about `CarbonContainer` or something?

Reply:
I want to rename our Carbon to Coder, the current Coder component to Editor, and then the Editor component to Carbon. What do you think?